tests/install/centos7/README.md

```diff
@@ -0,0 +1,46 @@
+RKE2 Install on CentOS 8
```

Comment:
All of theses READMEs are the same except for the OS that is being tested, can you make 1 document either under the `tests/install` folder or add a section to https://github.com/rancher/rke2/blob/master/developer-docs/testing.md

Reply:
You'll notice that the MicroOS has additional details. I expect these to diverge per OS over time and chose to not assume that someone looking to run some quick tests will know how to navigate our documentation.